BankManager/Sources/BankManager/Bank.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  File.swift
+//  
+//
+//  Created by Diana, Hamzzi on 3/20/24.
+//
+
+class Bank {
```

Comment:
static dispatch를 위해서 final 키워드를 붙여주면 좋을거 같습니다.
```suggestion
final class Bank {
```

Reply:
상속이 필요 없는 Class가 Dynamic Dispatch를 사용하면 성능상 손해라는 것을 확인하였습니다!
확인하고 수정하였습니다 ☺️